### PR TITLE
Update instructions and scripts to newest packer version add better Automation with scripts

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

File: scripts/vagrant.sh [hunk 1]
```diff
@@ -30,3 +30,11 @@ chown -R ${VAGRANT_USER}:${VAGRANT_USER} ${VAGRANT_HOME}/.ssh
 
 echo '==> Recording box config date'
 date > /etc/vagrant_box_build_time
+
+echo '==> Set DNS to known non interupting defaults'
+# See https://wiki.ipfire.org/dns/public-servers for details
+# Chosen servers are Digitalcourage e.V. and Cloudflare
+cat << EOF > /etc/resolv.conf
```
Comment (pyhalov) [inline scripts/vagrant.sh:37]: why not 8.8.8.8? ;)

Comment (Toasterson) [reply scripts/vagrant.sh:37]: AFAIK theoretically they aren't public. :) Just that nobody changed that yet in fear of breaking the web. In the ipfire link you can find the result of the firewall and router os vendor with the same name about how to address that fact. Google works as well but I can't remember if they ever stated an intent to make these servers publicly available.

**Comment by Toasterson** [comment]
Anything left to change or did this get lost in the gaps?

File: build.sh [hunk 2]
```diff
@@ -1,3 +1,31 @@
 #!/usr/bin/env bash
 
-packer build -parallel=false hipster.pkr.hcl
\ No newline at end of file
+set -ex
+
+BUILD_VERSION=$(date +%Y%m%d)
+
```
Comment (drscream) [inline build.sh]: Could we use some kind of option to select if only VirtualBox or qemu should be build. Maybe a Makefile would be a more suitable option?

Comment (Toasterson) [reply build.sh]: Actually a Makefile is a nice idea for that situation. I'll have a look into while crafting the next release (around next week) and add it.

Comment (Toasterson) [reply build.sh]: Added a makefile :)

**Comment by Toasterson** [comment]
nybody have some more wishes comments? Otherwise I'll merge as this has been the base for all vagrant box releases for more than half a year :)

File: hipster.pkr.hcl [hunk 3]
```diff
@@ -82,7 +87,7 @@ source "qemu" "oi-hipster" {
   disk_size = local.disk_size
   iso_checksum = var.iso_checksum
   iso_url =  local.iso_url
-  iso_checksum_type = local.iso_checksum_type
+  //iso_checksum_type = local.iso_checksum_type
   shutdown_command = local.shutdown_command
```
Comment (AndWac) [inline hipster.pkr.hcl:90]: Can you add a comment why it has been commented out? Either this or completely remove the lines.

**Review by AndWac — APPROVED** [review]
Please check whether you can squash some of the commits before merging the PR. You have more commits than files atm.

**Pull request merged**